draft-ietf-scitt-architecture.md

```diff
@@ -686,156 +572,67 @@ The last two steps may be shared between a batch of Signed Statements recorded i
 A Transparency Service MUST ensure that a Signed Statement is registered before releasing its Receipt, so that it can always back up the Receipt by releasing the corresponding entry (the now Transparent Statement) in the Append-only Log.
 Conversely, the Transparency Service MAY re-issue Receipts for the Append-only Log content, for instance after a transient fault during Signed Statement registration.
 
-## Receipts & Transparent Statements  {#Receipt}
+
+## Transparent Statements {#Receipt}
+
+The client (which is not necessarily the issuer) that registers a Signed Statement and receives a Receipt can produce a Transparent Statment by adding the Receipt to the Unprotected header of the Signed Statement.
```

Comment:
Discuss the possible issue on this issue!

Reply:
See #189